### DISCUSSION TITLE
Prácticas de CSS y Javascript

### DISCUSSION THREAD

**Discussion by JoseRoldan-555** [Launch-X-Latam/MisionFrontEnd#360] [Q&A]
Disculpen que practicas de css debemos de hacer y que práctica de Javascript debemos de acabar

**Comment by doguedogue** [comment]
Aca estan las carpetas de prácticas:

- CSS https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/03%20-%20CSS/practica
- JS https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/04%20-%20JS/practica
